### ISSUE TITLE
Incorrect border transform

### ISSUE THREAD

**Issue by pocketjoso** [jakubpawlowicz/clean-css#844]
Hi again,
Thanks as usual for all the hard work on maintaining and improving clean-css! Here's my problem:

# Version
I'm experiencing this on `3.4.22` (and all other 3.x versions), but I noted it did _not_ happen back in `3.1.9`.

# Input

``` css
input.button {
  border:2px
solid
}
```
# Result

```css
input.button {
  border:none
}
```
This is obviously problematic since this value was not part of the original css.

# Expected
Browsers seem to treat the input as:
```css
input.button {
  border:2px solid
}
```
But it also seems like a reasonable result to me to ignore the border declaration.

**Comment by jakubpawlowicz** [comment]
Thanks @pocketjoso for kind words!

I can see it happening in 3.4 but not in 3.1 or 4.0-pre. Stay tuned, fix is coming.

**Comment by jakubpawlowicz** [comment]
It's fixed in 3.4.23. Cheers!

**Comment by pocketjoso** [comment]
Awesome!
